libs/atqr/domain/src/lib/challenge-entity/challenge-entity.test.ts

```diff
@@ -0,0 +1,131 @@
+import { Challenge } from './challenge-entity';
+import { Player } from '../player';
+import { Guid } from '@tokilabs/lang/';
+import { EmailAddress } from '../..';
+import { PaymentMethodEntity, PaymentMethodEnum } from '../payment-method';
+
+// arrange, act and assert
+describe('Challenge', () => {
+  test('Create and call a new challenge', () => {
+    const email = new EmailAddress('fulano@hotmai.com');
+    const player = new Player('fulano', email);
+    const date = new Date(2022, 11, 12);
+    const creditCard = PaymentMethodEnum.creditCard;
+    const payment = new PaymentMethodEntity(creditCard, 'pagar.me', 'hau20283');
+
+    const a = new Challenge(
```

Comment:
Also, variable names should be given the same attention as tests descriptions. It is much simpler to use a when you're writing it for the first time or when making a scratch pass, but if someone else has to do maintenance on this test later it could become somewhat confusing.